src/threads/table.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { forumTable } from './table';
+
+describe('Forum table', () => {
+  const participantId = '12';
+  const itemId = '42';
+  const userId = '1001';
+
+  it('should add a thread event and retrieve new event in thread events', async () => {
+    await expect(forumTable.addThreadEvent(participantId, itemId, { type: 'thread_opened', byUserId: userId })).resolves.toBe(undefined);
+    await expect(forumTable.getThreadEvents(participantId, itemId)).resolves.toMatchObject([{
+      threadId: expect.any(String),
+      timestamp: expect.any(Number),
+      timeToLive: expect.any(Number),
+      type: 'thread_opened',
+      byUserId: userId,
+    }]);
+  });
```

Comment:
Even if it is better than nothing, testing a db by calling your own "add" and "get" in a same test is not a really valid way of testing.  

A proper way of testing these would be: 
- for `addThreadEvent`, empty the db, add an element, then dump the whole db and validate the content
- for `getThreadEvents`, seed the db with different content (expected and not expected) and each time check that what is returned by the function is what we want

Reply:
I see that `serverless` has a way to seed (`sls dynamodb seed`) the db, so it should not be complicated

Reply:
I am using jest here not sls, which uses under the hood dynamodb-local. I did it another way.
Fixed